termine.html

```diff
@@ -15,31 +15,6 @@
         </tr>
       </thead>
       <tbody>
-        <tr>
-          <td>04.05. – 06.05.2018</td>
-          <td>Essen</td>
-          <td>Workshop „Bewerbungstraining”</td>
-        </tr>
-        <tr>
-          <td>28.05.2018</td>
-          <td>16:15 Uhr P2-E0-414</td>
-          <td> Seminar: „Berufsfelder der Physik“, Dr. Andre Steffen (HELLA GmbH)</td>
-        </tr>
-        <tr>
-          <td>01.06. – 03.06.2018</td>
-          <td>Dortmund</td>
-          <td>Hackathon „Modern Web Services”</td>
-        </tr>
-        <tr>
-          <td>11.06.2018</td>
-          <td>16:15 Uhr P2-E0-414</td>
-          <td> Seminar: „Berufsfelder der Physik“, Dr. Peter Karbach (BAAINBw)</td>
-        </tr>
-        <tr>
-          <td>14.06.2018</td>
-          <td>ab 18 Uhr im Innenhof der Physik</td>
-          <td>Big Bang</td>
-        </tr>
```

Comment:
Maybe we could change the color to something grey here instead of removing these lines. It would be a nice and dense overview of our activities. We could even link to the reported posts.

Reply:
Yep that would work out nicely.